### PR TITLE
All Images Converted to Transparent PNG

### LINKED ISSUE #114: Fix Image Quality Issue across the Website
**Describe the issue**

Convert every image - up until, Page 13 to
 - A transparent ```png``` version of the image.
 - A folder name consisting of the page number, along with the new ```png format``` transparent images for the dishes.
 - Convert the image size to under 500kb, with no hit to image resolution.

### PR DESCRIPTION
**Pull request by bhavkushwaha**
## Submission Checklist ✅
Open a pull request only after you have finished adding your recipes. Please complete the below checklist. Failure to follow these instructions may result in your pull request being closed without review, and may affect your contribution.

**Important**: *Reviewers will review your pull requests when applicable, based on a queue. Not starring the repo, will lead to questions*

-   [X] I have starred and forked the repo
-   [X] I read the instructions to submit the page.
-   [X] I have successfully converted all the Images of the Project into transparent PNG format.
-   [X] I have also organised all the food images of all the pages in folders
-   [X] I have also replaced better images if necessary.
-   [X] I have also made the requied changes in the html files to incorporate the new images and folders.
-   [X] I have added a `reviewer` to this pull request.
-   [X] I understand that a reviewer will merge my pull request after review, and that I must comply to recieving code reviews.

### CODE REVIEW

File: images/Page 11/breadpizza.png
Comment (TheGuyDangerous) [inline images/Page 11/breadpizza.png:1]: the left plate border is overflowing, please fix it

File: images/Page 11/kaju katli.png
Comment (TheGuyDangerous) [inline images/Page 11/kaju katli.png:1]: the ```upper, and right plate borders``` are overflowing, please fix it

File: images/Page 12/bhel-puri.png
Comment (TheGuyDangerous) [inline images/Page 12/bhel-puri.png:1]: the ```left and right plate borders``` are overflowing, please fix it

File: images/Page 12/khichdi-masala.png
Comment (TheGuyDangerous) [inline images/Page 12/khichdi-masala.png:1]: the ```bottom plate border``` is overflowing, please fix it

File: images/Page 4/malai-kofta.png
Comment (TheGuyDangerous) [inline images/Page 4/malai-kofta.png:1]: ```all the plate borders``` are overflowing, please fix it

File: images/Page 6/burger.png
Comment (TheGuyDangerous) [inline images/Page 6/burger.png:1]: the ```left and plate borders``` are overflowing, please fix it

File: images/Page 6/chicken-shawarma.png
Comment (TheGuyDangerous) [inline images/Page 6/chicken-shawarma.png:1]: the ```right plate border``` is overflowing, please fix it

File: images/Page 7/Peri-Peri-Cheese-Balls.png
Comment (TheGuyDangerous) [inline images/Page 7/Peri-Peri-Cheese-Balls.png:1]: the ```right, bottom, and left plate borders``` are overflowing, please fix it

File: images/Page 7/murgh-masala.png
Comment (TheGuyDangerous) [inline images/Page 7/murgh-masala.png:1]: the ```right plate border``` is overflowing, please fix it

File: images/Page 9/alfredo.png
Comment (TheGuyDangerous) [inline images/Page 9/alfredo.png:1]: the ```top and bottom plate border``` are overflowing, please fix it

File: images/Page 11/cutlet.png
Comment (TheGuyDangerous) [inline images/Page 11/cutlet.png:1]: the ```upper, and bottom plate borders``` are overflowing, please fix it

File: images/Page 10/choclate-cake.png
Comment (TheGuyDangerous) [inline images/Page 10/choclate-cake.png:1]: the ```left, and right plate borders``` are overflowing, please fix it

File: images/Page 10/honey-chilli.png
Comment (TheGuyDangerous) [inline images/Page 10/honey-chilli.png:1]: ```all the plate borders``` are overflowing, please fix it

File: images/Page 10/moong-dal-halwa.png
Comment (TheGuyDangerous) [inline images/Page 10/moong-dal-halwa.png:1]: the ```left, and right plate borders``` are overflowing, please fix it

File: images/Page 11/thekua.png
Comment (TheGuyDangerous) [inline images/Page 11/thekua.png:1]: the ```bottom plate borders``` are overflowing, please fix it

**Review by TheGuyDangerous — CHANGES_REQUESTED** [review]

**Comment by TheGuyDangerous** [comment]
@bhavkushwaha please fix all the border issues for the above mentioned files, and you're all set !

**Comment by bhavkushwaha** [comment]
Hey @TheGuyDangerous I have modified all the mentioned Images as per instructed, hope they meet the requirements now.

**Review by TheGuyDangerous — APPROVED** [review]

**Comment by TheGuyDangerous** [comment]
Great Job and nicely done !! ✅

**Pull request merged**